src/features/content-explorer/content-explorer/ContentExplorer.scss

```diff
@@ -71,6 +71,33 @@
         }
     }
 
+    .bdl-ContentExplorerIncludeSubfolders {
+        display: flex;
+        align-items: center;
+        justify-content: space-between;
+        height: 10 * $bdl-grid-unit;
+        margin-top: 20px;
+        margin-bottom: -10px;
```

Comment:
why do we need the negative margin? should the margin-top be decreased?

Reply:
The negative margin was added because the base toggle and checkbox components had a bottom margin of 20px on them. So a negative margin for the outer div helped push everything down to where it should be positioned

Reply:
instead of the negative margin, could we target the toggle and checkbox classes and reset those within this block?

```
.bdl-ContentExplorerIncludeSubfolders {
    .checkbox-container ,
    .toggle-container {
        margin: 0;
    }
}
```

Reply:
Yes this is much better